public/docs/ts/latest/guide/production-deploy.jade

```diff
@@ -0,0 +1,480 @@
+include ../_util-fns
+  
+:marked
+  This page describes tools and techniques for optimizing your Angular application and deploying it to production.
+
+a#toc
+:marked
+  ## Table of contents
+  * [Overview](#overview)
+  * [Simplest deployment possible](#dev-deploy)
+  * [Optimize for production](#optimize)
+    * [Ahead-of-Time (AOT) compilation](#aot)
+    * [Webpack](#webpack)
+    * [Tree shaking with _rollup_](#rollup)
+    * [Measure performance first](#measure)
+  * [Angular configuration](#angular-configuration)
+    * [The `base` tag](#base-tag)
+    * [Enable production mode](#enableprodmode)
+    * [Lazy loading](#lazy-loading)
+  * [Server configuration](#server-configuration)
+    * [Routed apps must fallback to `index.html`](#fallback)  
+    * [CORS: requesting services from a different server](#cors)
+
+a#overview
+.l-main-section
+:marked
+  ## Overview
+
+  This guide describes techniques for preparing and deploying an Angular application to a server running remotely.
+  The techniques progress from _easy but suboptimal_ to _more optimal and more involved_.
+
+  * The [simple way](#dev-deploy "Simplest deployment possible") is to copy the development environment to the server.
```

Comment:
There is value is describing how to deploy your dev code with the minimum set of changes. This is a question that comes up a couple of times every week.

I'll add a note that this isn't recommended per se, but rather a fallback.